instrumentation/kamon-pekko-http/src/main/scala-3/kamon/instrumentation/pekko/http/PekkoHttpServerInstrumentation.scala

```diff
@@ -0,0 +1,323 @@
+package kamon.instrumentation.pekko.http
```

Comment:
I think this file can/should be deduplicated between scala versions now - it's essentially the same, modulo formatting, for 2.12/2.13/3

Reply:
the scala-3 version is explicitly handing the Tuple in line 290.
the implicit way didn't compile

Reply:
Don't think it's worth having three copies just to avoid that in the other two copies 😄 Shouldn't cause an error with them